src/css/core.scss

```
@@ -322,8 +322,10 @@ nav {
 		height: 27px;
 	}
```

Even though these elements are not visible, they still react to mouse events (such as hovers). Since the intention is that they should not be visible unless the heading is hovered over, or if the heading/sub-headings are tabbed to, then we should disable mouse events on them until they are visible.

```suggestion
		pointer-events: none;
	}
```